packages/ui/src/components/avatar/stx20-avatar-icon.web.tsx

```diff
@@ -1,4 +1,5 @@
-import Stx20AvatarIconSrc from '../../assets-web/avatars/stx20-avatar-icon.png';
+import Stx20AvatarIconSrc from 'src/assets-web/avatars/stx20-avatar-icon.png';
```

Comment:
Trying this as I can't see it loading when testing locally 
![Screenshot 2024-06-19 at 14 10 16](https://github.com/leather-wallet/mono/assets/2938440/04e29530-25df-42f6-9576-9b55ee82843d)